quantlib/__init__.py

```diff
@@ -0,0 +1,6 @@
+import sys
+import os
+
+# add the root folder to PATH so the quantlib dll can be found
+if sys.platform == 'win32':
```

Comment:
@tonyroberts there is another approach to this which feels cleaner to me than changing the PATH. Using ctypes, you can directly load the dll's in memory. 

```
from ctypes import cdll
HERE = dirname(abspath(__file__))

dlls = [get the list of QL dll's]

for dll in dlls:
    cdll.LoadLibrary(dll)
```